Zilon.Core/Zilon.Bot.Players.DevelopmentTests/AutoplayEngine.cs

```diff
@@ -22,11 +22,13 @@ protected override void ConfigBotAux()
 
         protected override void CatchActorTaskExecutionException(ActorTaskExecutionException exception)
         {
+            Console.WriteLine(exception);
             throw exception;
         }
 
         protected override void CatchException(Exception exception)
         {
+            Console.WriteLine(exception);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310821)